JumpscaleBuildersCommunity/monitoring/BuilderGrafanaAdmin.py

```diff
@@ -0,0 +1,72 @@
+from Jumpscale import j
+
+builder_method = j.baseclasses.builder_method
+
+
+class BuilderGrafanaAdmin(j.baseclasses.builder):
+
+    __jslocation__ = "j.builders.monitoring.grafana_admin"
+
+    def _init(self, **kwargs):
+        j.core.tools.dir_ensure(self.DIR_BUILD)
+        self.version = "7.0.3"
+        self.DIR_BASE_GRAFANA = "/sandbox/grafana"
```

Comment:
maybe make the usage of {DIR_BASE} j.core.dirs.base_dir instead of hardcoded /sandbox